python/whylogs/api/store/sqlite_store.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import sqlite3
+from datetime import timedelta, timezone
+
+from whylogs.api.store import ProfileStore
+from whylogs.api.store.query import BaseQuery, DatasetIdQuery, DateQuery
+from whylogs.core import DatasetProfile, DatasetProfileView
+
+logger = logging.getLogger(__name__)
+
+
+class SQLiteStore(ProfileStore):
+    def __init__(self):
+        self._db_location = os.getenv("SQLITE_STORE_LOCATION")
+        if self._db_location is None:
+            raise ValueError("You must define a store with the SQLITE_STORE_LOCATION env var")
```

Comment:
We should have a task to centralize all of the config logic behind a single class/interface. Not crucial atm but not having it definitely leads to code duplication, difficulty knowing which parameters are required/exist, and difficulty mocking/testing things when they depend directly on the environment.

Reply:
my idea is to leave this to the Profile Store Service only, at least for now, where a config file will be very useful and I can also see more stuff on whylogs in general - like constraints definition - that could use something like a YAML or w/e, so I would prefer to build something at once then

Reply:
my idea is to have it all under a `config.ini`, but only on the profile-store-service, as it's where it will actually make sense to use, since other stuff also require env vars -- such as the s3writer class we rely on for the service.